examples/demo-jobs/.neuro/batch-action.yml

```diff
@@ -12,7 +12,7 @@ outputs:
   needs: ["task_1_${{ inputs.arg1 }}_second", task_2]
   res1:
     descr: action result 1
-    value: ${{ to_json(needs) }}
+    value: ${{ fmt('task_1_', 'inputs.arg1', '_second') }}.needs.task1 task_2.needs.task2
```

Comment:
I'm completely lost here: was this fmt() call is doing? Is the following correct translation to python?
```
>>> 'task_'.format('inputs.arg1', '_second')
'task_'
```

Reply:
Uups. My wrong.
The initial problem was that the `needs` context has no stable order.
The update has the fixed expression